### PR TITLE
TA-1311: Test implementation

### PR DESCRIPTION
**Pull request by EvgeniiNikonets**

### CODE REVIEW

File: test_files/rate_limit_event_condition.json [hunk 1]
```diff
@@ -0,0 +1,1157 @@
...
+                "Value": "[\n{\n\"attrs\": \"company\"\n}\n]"
+              },
+              {
+                "Key": "action",
+                "Value": "\"action-rate-limit-block\""
+              },
+              {
+                "Key": "pairwith",
+                "Value": "{\n\"self\":\"self\"\n}"
+              }
```
Comment (olgasobolevsky) [inline test_files/rate_limit_event_condition.json:884]: Here should be :
{
                "Key": "pairwith",
                "Value": "{\n\"attrs\":\"uri\"\n}"
              }

File: test_files/rate_limit_event_condition.json [hunk 2]
```diff
@@ -0,0 +1,1157 @@
+{
+  "Name": "Rate Limit - Event condition TA-1311",
+  "Skip": false,
```
Comment (olgasobolevsky) [inline test_files/rate_limit_event_condition.json:3]: This suite should be reorganized to save on Publish times.
1. Create rate limit with Event by Header
2. Create rate limit with Event by Cookie
3. Create rate limit with Event by Argument
4. Create rate limit with Event by Attribute:URI
5. Create security policy with 4 different path mappings and add rate limits to different path mappings
6. Send requests to test each rate limit according to path
7. Check logs for each rate limit, filter by path

File: test_files/rate_limit_event_condition.json (hunk 1 above)
Comment (EvgeniiNikonets) [reply test_files/rate_limit_event_condition.json:884]: i copied it from our reqest on planet

Comment (olgasobolevsky) [reply test_files/rate_limit_event_condition.json:884]: attr-uri

File: test_files/rate_limit_event_condition.json [hunk 3]
```diff
@@ -0,0 +1,1157 @@
...
+                "Value": "10"
+              },
+              {
+                "Key": "threshold",
+                "Value": "2"
+              },
+              {
+                "Key": "key",
+                "Value": "[\n{\n\"attrs\": \"company\"\n}\n]"
+              },
```
Comment (olgasobolevsky) [inline test_files/rate_limit_event_condition.json:47]: The Count by for all rate limits in this test suite is Header:org
{
                "Key": "key",
                "Value": "[\n{\n\"headers\": \"org\"\n}\n]"
              }
count by -header-org

File: test_files/rate_limit_event_condition.json (hunk 1 above)
Comment (olgasobolevsky) [reply test_files/rate_limit_event_condition.json:884]: if this is rate limit for https://reblaze.atlassian.net/browse/TA-1311 Step 7 
attr-uri

File: test_files/rate_limit_event_condition.json [hunk 4]
```diff
@@ -0,0 +1,1039 @@
...
+            "Code": "471",
+            "Type": "content",
+            "Single": true,
+            "Text": "b''"
+          }
+        },
+        {
+          "ID": "wait-for-timeout-10sec",
+          "Skip": false
+        },
```
Comment (olgasobolevsky) [inline test_files/rate_limit_event_condition.json:691]: This wait step is redundant cause in next step you are sending traffic to another path.

**Review by olgasobolevsky — APPROVED** [review]

**Pull request merged**